### PR TITLE
Feature: workshop dashboard

### PR DESCRIPTION
**Pull request by kamilsdz**
I added select field for dashboard. We should add it to Users dashboard too (AdminUser / RegularUser).

### CODE REVIEW

File: app/dashboards/workshop_dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    kind: Field::SelectBasic.with_options(choices:
+      %w[
```
Comment (fab-ian) [inline app/dashboards/workshop_dashboard.rb]: Please fetch data using method 'kinds' of model Workshop (Workshop.kinds)

File: app/dashboards/workshop_dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  FORM_ATTRIBUTES = %i[
+    kind
+    latitude
+    longitude
+  ].freeze
+
+  # Overwrite this method to customize how workshops are displayed
+  # across all pages of the admin dashboard.
+  #
+  # def display_resource(workshop)
```
Comment (fab-ian) [inline app/dashboards/workshop_dashboard.rb]: Uncomment and adjust this method (display_resource)

File: app/dashboards/workshop_dashboard.rb (hunk 1 above)
Comment (kamilsdz) [reply app/dashboards/workshop_dashboard.rb]: Okay Fabian, but why kind should be plural? Names of table fields are singular.

Comment (fab-ian) [reply app/dashboards/workshop_dashboard.rb]: If you want fetch a list of defined enum you use plural name of field (kinds):

```
2.4.2 :002 > ap Workshop.kinds
{
                    "audi" => 0,
                 "citroen" => 1,
                    "fiat" => 2,
                   "honda" => 3,
                 "hyundai" => 4,
                   "mazda" => 5,
                    "opel" => 6,
                 "peugeot" => 7,
                 "renault" => 8,
                  "toyota" => 9,
              "volkswagen" => 10,
    "warsztat_lakierniczy" => 11,
    "warsztat_samochodowy" => 12
}
 => nil 
2.4.2 :003 >
```
Or in your case you can probably use: 
```
2.4.2 :004 > Workshop.kinds.keys
 => ["audi", "citroen", "fiat", "honda", "hyundai", "mazda", "opel", "peugeot", "renault", "toyota", "volkswagen", "warsztat_lakierniczy", "warsztat_samochodowy"] 
2.4.2 :005 >
```

Comment (fab-ian) [reply app/dashboards/workshop_dashboard.rb]: Something like this...
``` kind: Field::SelectBasic.with_options(choices: Workshop.kinds.keys) ```

File: app/dashboards/workshop_dashboard.rb [hunk 3]
```diff
@@ -28,7 +28,7 @@ class WorkshopDashboard < Administrate::BaseDashboard
   # By default, it's limited to four items to reduce clutter on index pages.
   # Feel free to add, remove, or rearrange items.
   COLLECTION_ATTRIBUTES = %i[
-    kind
+    kinds
```
Comment (fab-ian) [inline app/dashboards/workshop_dashboard.rb]: Please rollback migration (rake db:rollback) The field should be named 'kind'.

**Pull request merged**